project-setup

```diff
@@ -0,0 +1,181 @@
+
+        if [ $python_version != "3.7" ] ; then 
+            echo "Please install Python version 3.7.2 or greater"
+            exit 1
+        fi
+    fi
+fi
+
+echo "Found Python 3.7 🐍"
+echo ""
+
+if ! [ -x "$(command -v pipenv)" ]; then
+
+    read -p "You do not have pipenv installed. The default installation method will be Pip. Would you like to install it with [P]ip or [H]omebrew? " choice
+
+    if [ $choice == "h" ] || [ $choice == "H" ] ; then
+        if ! [ -x "$(command -v brew)" ] ; then
+            echo "🛑 We could not find Homebrew installed in your system. " \
+                "Please take a look on how to install it here: https://brew.sh/ 🛑"
+            exit 1
+        fi
+
+        echo "Found Homebre 🍺. Installing Pipenv."
+        brew install pipenv
+    else
+        echo "Installing using Pip"
+
+        if [ -x "$(command -v pip3.7)" ]; then
+            pip3.7 install pipenv
+        elif [ -x "$(command -v pip3)" ]; then
+            pip3 install pipenv
+        else
+            echo "🛑 We could not find pip installed in your system. 🛑"
+            exit 1
+        fi
+    fi
+fi
+
+pipenv install --dev
+
+pipenv run pre-commit install
+
+echo ""
+
+echo "Creating the .env file. Remember to change this file with your settings."
+if [ -f example.env ]; then
+    cp example.env .env
+else
+    echo "PYTHONUNBUFFERED=1" >> .env
+    echo "" >> .env
+    echo "DJANGO_SECRET_KEY=dont-tell-eve" >> .env
+    echo "DATABASE_URL=postgres://postgres:password@database/postgres" >> .env
+    echo "REDIS_URL=redis://redis:6379" >> .env
+    echo "FILEPREVIEWS_API_KEY=""" >> .env
+    echo "FILEPREVIEWS_API_SECRET=""" >> .env
+fi
+
+database_backup_dump="`dirname "$0"`/docker/postgres/backup.dump"
+
+if ! [ -f $database_backup_dump ] ; then
+    echo ""
+    echo "We need to download our development database dump. This file is about 100MB. 😱"
+    read -p "Would you still want to download this? [Y/n] " choice
+
+    if [ $choice == "Y" ] || [ $choice == "y" ] ; then
+        curl -o $database_backup_dump https://s3.amazonaws.com/data.contratospr.com/2019-08-24/backup.dump
+    else
+        echo "The file will not be downloaded. If you change your mind you can download it here: " \
+            "https://s3.amazonaws.com/data.contratospr.com/2019-08-24/backup.dump"
+        echo "⚠️ Remember that if you run this project without this you will not have any test data. ⚠️"
+    fi
+fi
+
+echo ""
+echo "You're all set! When ever you're ready to run this project using Docker and docker-compose."
+read -p "Would you like to run these commands now? [Y/n] " choice
+
+if [ $choice == "Y" ] || [ $choice == "y" ] ; then
+    echo ""
+    read -p "Do you want to run this in detached mode (will not show logs)? [Y/n]" choice
+    echo ""
+
+    echo "Building Docker images. 🐳🐳🐳"
+
+    if [ $choice == "Y" ] || [ $choice == "y" ] ; then
+        docker-compose up --build -d
+    else
+        docker-compose up --build
```

Comment:
If running in the foreground, will the rest of this script execute?

Reply:
Excellent point. I think it should just be removed.